src/transformers/models/grounding_dino/processing_grounding_dino.py

```diff
@@ -83,21 +111,8 @@ def __call__(
         self,
         images: ImageInput = None,
         text: Union[TextInput, PreTokenizedInput, List[TextInput], List[PreTokenizedInput]] = None,
```

Comment:
FYI the call method now looks uniformly like this:
```
text: Union[TextInput, PreTokenizedInput, List[TextInput], List[PreTokenizedInput]] = None,
audio=None,
videos=None,
**kwargs: Unpack[GroundingDinoProcessorKwargs],
```
I don't see the video keyword argument here so that would need to be added

Reply:
@NielsRogge I think we can just set audio and videos to none in this project but let's see what he says.

Reply:
I think it's acceptable to set it to None and then to not advertise it in the docstring